relay/network_graph/alg.py

```diff
@@ -87,7 +88,10 @@ def _least_cost_path_helper(
         if cost_from_start_to_node > least_costs[node]:
```

Comment:
I would guess this line is redundant now

Reply:
no, we may still have put an element with non-minimal cost onto the queue and we don't want to look at them again.